File: Resueltos_por_competidor/Santiago_Ortiz/NearlyLuckyNumber.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: Nearly Lucky Number
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/110/A
 */

public class NearlyLuckyNumber {

	static int contador = 0;

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		long n = sc.nextLong();
		boolean lucky = true;

		String ns = Long.toString(n);
		for (int i = 0; i < ns.length(); i++) {
			char c = ns.charAt(i);
			if (c == '4' || c == '7') {
				contador++;
			}
		}
		if(contador == 4 || contador == 7) {
			System.out.println("YES");
		} else {
			System.out.println("NO");
		}
	}

}

File: Resueltos_por_competidor/Martin_Peña/NearlyLuckyNumber.cpp
/*
* Autor: pgms1103-afk
* Problema: A. Nearly Lucky Number
* Juez Online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/110/A
*/
#include <bits/stdc++.h>

using namespace std;

int main()
{
    string numero;
    cin >> numero;

    int contador = 0;
    for(int i = 0; i<numero.length(); i++){
        if(numero[i] == '4' || numero[i] == '7'){
            contador += 1;
        }
    }
    if(contador == 4 || contador == 7){
        cout << "YES" << endl;
    }else{
        cout << "NO" << endl;
    }

    return 0;
}

